pkg/controllers/provisioning/v1alpha1/util/util.go

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I like that you're sharing this for both controllers -- I don't like that it's called "util". As is, this is an indeterminitely large function bag without encapsulation. Minimally, we could do something like /util/scheduling/pod.go

scheduling.IsNotIgnored
scheduling.IsUnschedulable

Reply:
While I don't envision us using that many util functions that span both allocator and reallocator that don't all include scheduling (very possible chance I'm wrong), I like separating it into a scheduling folder.

Reply:
We can always revisit this and collapse if necessary.